update atomicdex sidebar menu

smk762 opened the pull request:
closes https://github.com/KomodoPlatform/developer-docs/issues/380

artemii235 commented on docs/.vuepress/atomicDEX-sidebar.js:
```diff
@@ -115,130 +115,102 @@ let atomicDEXsidebar = {
+          ["/basic-docs/atomicdex-api-legacy/orderbook_depth.md", "orderbook_depth"],
+          ["/basic-docs/atomicdex-api-legacy/orders_history_by_filter.md", "orders_history_by_filter"],
+          ["/basic-docs/atomicdex-api-legacy/order_status.md", "order_status"],
+          ["/basic-docs/atomicdex-api-legacy/my_orders.md", "my_orders"],
```
`my_orders` should be right after `cancel_order` :)

smk762 replied:
:facepalm: I need to go back to kindergarten :sweat_smile:

artemii235 commented on docs/.vuepress/atomicDEX-sidebar.js:
```diff
@@ -282,95 +269,113 @@ let atomicDEXsidebar = {
+          title: "Utility",
+          collapsable: true,
+          children: [
+            ["/basic-docs/atomicdex-api-20-dev/recreate_swap_data.md", "recreate_swap_data"],
```
I think these can be moved to `Swaps` section.

smk762 replied:
There was no "swaps" subsection in the 2.0 sections, so I renamed "Utility" to "Swaps". In the legacy section, I also moved swaps related methods from utility subsection to swaps subsection.

artemii235 reviewed (changes requested):
Thanks for the PR! A couple of notes.

smk762 commented:
@artemii235 I've also added a table of available methods to this PR
https://github.com/KomodoPlatform/developer-docs/blob/66b000ef06b2476c70e28d20fc275c315fe97cfd/docs/basic-docs/atomicdex/atomicdex-methods.md

Initial intent was to track what is missing or needs to move to v2 in Desktop GUI, though I figured it might be a good reference for others also.

With the latest release, I'll need to review what has moved from dev into master and update the table/sidebar accordingly.

smk762 commented:
I've moved all methods now in master to their appropriate position within the sidebar menus and table, and updated urls within pointing to the older reference doc locations.

artemii235 reviewed (approved):
👍
